.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -155,8 +154,8 @@ jobs:
 
   fast-tests:
     name: Fast CPU tests
-    if: github.event_name != 'pull_request' ||
-      (github.event_name == 'pull_request' && needs.setup-instance.result != 'skipped')
+    if: github.event_name != 'pull_request_target' ||
+      (github.event_name == 'pull_request_target' && needs.setup-instance.result != 'skipped')
     needs: [ should-run, setup-instance ]
```

Comment:
does this still need should run if it passes by setup instance ?

Reply:
You're right.